languages/csharp/sensitivity/varargs/varargs.tmt

```diff
@@ -0,0 +1,86 @@
+~extensions start~
+~extensions import *~
+~extensions end~
+
+~main body start~
+
+~helper functions start~
+
+~function get_first_string start~
+public string getFirstString(params string[] lines) {
+    return getStringWithIndex(0, lines);
+}
+~function end~
+
+~function get_string_with_index start~
+public string getStringWithIndex(int ind, params string[] lines) {
+    return lines[ind];
+}
+~function end~
+
+~function get_first_string_from_array start~
+public String getFirstStringFromArray(params string[] lines) {
+    return lines.ToList().ElementAt(0);
+}
+~function end~
+
+~function varargs_with_generics start~
+public <T> T varargsWithGenerics(params T[] elements) {
```

Comment:
Invalid position for generic, try `public T varargsWithGenerics<T>(params T[] elements) {`